agoradatatools/etl/transform.py

```diff
@@ -46,17 +46,20 @@ def rename_columns(df: pd.DataFrame, column_map: dict) -> pd.DataFrame:
     return df
 
 
-def nest_fields(df: pd.DataFrame, grouping: str, new_column: str) -> pd.DataFrame:
+def nest_fields(df: pd.DataFrame, grouping: str, new_column: str, drop_columns: list = []) -> pd.DataFrame:
     """
```

Comment:
Any idea why changes you merged in from dev are showing up as changes in this PR? Do I have the wrong view options selected or something?

Reply:
No, I tried to rebase my branch on dev and it went sideways. I'm going to close this PR and open a new one shortly...